FastMember/ObjectReader.cs

```diff
@@ -26,20 +38,21 @@ public class ObjectReader : DbDataReader
         /// <param name="members">The members that should be exposed to the reader</param>
         public static ObjectReader Create<T>(IEnumerable<T> source, params string[] members)
         {
-            return new ObjectReader(typeof(T), source, members);
+            return new ObjectReader(typeof(T), source, members: members);
         }
 
         /// <summary>
         /// Creates a new ObjectReader instance for reading the supplied data
         /// </summary>
         /// <param name="type">The expected Type of the information to be read</param>
         /// <param name="source">The sequence of objects to represent</param>
+        /// <param name="keyPropertyName">Current object property name to set IsKey column value for schema table</param>
         /// <param name="members">The members that should be exposed to the reader</param>
-        public ObjectReader(Type type, IEnumerable source, params string[] members)

```

Comment:
This is a hard break; we can't do that; whatever API changes must be binary compatible; I'm less concerned about build compatible, but that would be nice too.

Adding IsKey itself: fine

Reply:
I updated the pull request and made fixes for backwards compatibility
![image](https://user-images.githubusercontent.com/59282770/159429609-bc78ac99-e088-4efa-a030-1114a1d29188.png)

Reply:
I updated the pull request, but that didn't fix the problem. I will try to change my approach

Reply:
key property can be selected via expression
![image](https://user-images.githubusercontent.com/59282770/159440273-c0feb6f0-8d43-41de-8e59-b9ec0a4339c3.png)

Reply:
I create another constructor for backwards compatibility
![image](https://user-images.githubusercontent.com/59282770/159443903-7bd456df-ebb8-446b-b0f2-2b112d181459.png)